Add TypeScript support (#2117)
ViewComponent/view_component: docs/CHANGELOG.md

## main

> * Add typescript support to stimulus generator + fix bug not adding stimulus controller to erb when stimulus is activated by default.
> 
>     *Denis Pasin*
> 

Reviewer: ```suggestion
* Fix bug where stimulus controller was not added to ERB when stimulus was activated by default.

    *Denis Pasin*

* Add typescript support to stimulus generator.

    *Denis Pasin*

```
